README.md

```diff
@@ -45,10 +45,20 @@ the [the full documentation](https://isso-comments.de/docs/).
 
 ## Docker
 
-A Docker image with the latest stable release is provided at
-`ghcr.io/isso-comments/isso:latest`. See
+> [!NOTE]  
+> The Docker image tagging scheme for stable relases was changed from `:latest`
```

Comment:
I don't think this was ever actually true, so it doesn't seem useful to document that as previous behaviour.

Reply:
See above, that was the tagging policy I implemented and which was previously documented as such.

> A Docker image with the latest stable release is provided at `ghcr.io/isso-comments/isso:latest` [...]

It seems at least one user already relied on this, so I deem it prudent to document this change rather than silently `s/:latest/:release` in the docs.